backend/migrations/20201009200808_statusUpdates.js

```diff
@@ -0,0 +1,16 @@
+const { tableStatusUpdates } = require('../src/models/statusUpdate')
+
+exports.up = (knex) => {
+  return knex.schema.createTable(tableStatusUpdates, (table) => {
+    table.increments('StatusUpdateID').primary() // ToDo StatusID vs ID (postgres issue). Figure out uuid
```

Comment:
Remove this TODO comment.
```suggestion
    table.increments('StatusUpdateID').primary()
```